python/shotgun_model/data_handler_nav.py

```diff
@@ -146,10 +152,40 @@ class and the data payload from Shotgun should be provided via the
         num_modifications = 0
 
         new_uids = set()
+
+        # a list of sg data dicts to display items for
+        child_data = []
+
+        if item_path == self.__root_path:
+            self._log_debug("This is the root of the tree.")
+            parent_uid = None
+
+            if self.__include_root:
```

Comment:
The home button is supposed to take you some place that makes sense for your current context.

- If you are on a entity or task context, it should take you to an entity.
- If you a project context, it should take you to the project

I would have expected the Home button to take me to the project with your changes. The reason why it might not have done what I expected is because I didn't write the code correctly originally. :)

On [this line](https://github.com/shotgunsoftware/tk-multi-loader2/blob/develop/zero_config/python/tk_multi_loader/dialog.py#L828), I implemented the home button behaviour for the hierarchy.

It should probably have read something like `ctx.entity or ctx.project`. What happens if you pick that?